bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1414,9 +1414,10 @@ namespace JsRTApiTest
     static void CALLBACK PromiseContinuationCallback(JsValueRef task, void *callbackState)
     {
         CHECK(callbackState != nullptr);
-        CHECK(*(void **)callbackState == nullptr);
-        // All the task need to finish async, so we need to save the callback.
-        *(void **)callbackState = task;
+        CHECK(*(JsValueRef *) callbackState == JS_INVALID_REFERENCE);
```

Comment:
I feel like this was more correct before.  callback state is a void pointer so you don't need to cast it to assign the JsValueRef to it.